frontend/webpack.config.ts

```diff
@@ -68,6 +70,12 @@ const config: Configuration = {
               workers: require('os').cpus().length - 1,
             },
           },
+          isDevelopment && {
```

Comment:
This looks like the cause of the error. Won't this add `false` to the array when not in development mode?